drivers/serial/uart_nrfx_uarte.c

```diff
@@ -1831,14 +1827,14 @@ static void uarte_nrfx_set_power_state(const struct device *dev,
 				       enum pm_device_state new_state)
 {
 	NRF_UARTE_Type *uarte = get_uarte_instance(dev);
+#if defined(CONFIG_UART_ASYNC_API) || defined(UARTE_INTERRUPT_DRIVEN)
```

Comment:
Is this part of this change ?

Reply:
yes, previous to the removal of local states pm_control used `data` unconditionally (to get state, which was always available).